tests/test_utils.py

```diff
@@ -70,26 +75,35 @@ def callback_on_failure():
     return False
 
 
+class MockRequestModel(BaseModel):
+    message_id: str
+    test: bool = True
+
+
 class MqCallbackDecoratorClass:
     from neon_mq_connector.utils.rabbit_utils import create_mq_callback
     class_callback = Mock()
 
     def __init__(self):
         self.callback = Mock()
 
-    @create_mq_callback()
+    @create_mq_callback
```

Comment:
This looks like a breaking change to existing syntax. I'm not opposed, but I think we should release [0.8.0](https://github.com/NeonGeckoCom/neon_mq_connector/pull/122) and then push this to a 1.0.0 release

Reply:
> This looks like a breaking change to existing syntax. I'm not opposed, but I think we should release [0.8.0](https://github.com/NeonGeckoCom/neon_mq_connector/pull/122) and then push this to a 1.0.0 release

Old-fashioned way is going to work either, I just made an extra wrapper so we are not forced to explicitly call it with empty brackets